SHIBBOLETH_SP.md

```diff
@@ -6,27 +6,45 @@ category: authentication
 
 This document describes installing Shibboleth on Debian 9.
 
+## Requirements
+
+1. Apache
+
+        Apache webserver is required for Shibboleth SP.
```

Comment:
this seems not necessary to mention this as eduVPN installation makes sure Apache is configured and installed already...

Reply:
i have removed it.